AUTHORS

```diff
@@ -4,4 +4,31 @@
 # especially since many employees of one corporation may be contributing.
 # To see the full list of contributors, see the revision history in
 # source control.
-Google LLC
```

Comment:
> Create a top-level file named AUTHORS, which will identify the copyright holders of the project. For contributions made under a Corporate CLA, then the copyright belongs to that organization. If it was made under an Individual CLA, the submitter has indicated that they own the copyright.

Reading the guidelines: https://opensource.google/docs/releasing/authors/.
It seems that when copyright belongs to the company we should not list individual contributors' names.

I see your concern that, for KFP v0.1, the records are not included in public git history. I think we should mark that they all worked for Google and the copyright for `KFP` v0.1 belongs to Google.

Reply:
+1.  Everyone listed here can be considered under `Google LLC` as suggested. I don't think there's a need to have contributors by version, since that wouldn't be scalable. I would also suggest not adding this file unless someone requests explicitly to be added to the AUTHORS file, as suggested in the guideline:

> It is not necessary to proactively list the contributor for every single contribution you receive. Rather, it is sufficient to add contributors when they request to be added.

Reply:
>I don't think there's a need to have contributors by version, since that wouldn't be scalable.

I agree. The only exception is for the 0.1 version due to the removal of history when moving to the new repo. I also wanted to mention Nikhil Kothari who led this project initially.

I've added a single list of notable contributors in chronological order. This seems saleable.

Reply:
Thanks Alexey. However, the criteria of 5+ here seems like an arbitrary number. Why not 1, or 10? Again, I think this file is not maintainable or scalable. Let's do the following instead - use `Google LLC` for all Googlers (so there's no need for a 0.1 section or explicit list of Googlers) and additionally mention the following in the doc:
* a paragraph explaining Kubeflow Authors includes everyone who has ever contributed to the repo
* if external contributors wish to be explicitly included in the file, they can send a PR adding themselves

A link to the guidelines in https://opensource.google/docs/releasing/authors/ would also be appropriate.

Reply:
I don't see why we need to deviate from the guidelines. I would add to the AUTHORS file when there is a request from someone on their own behalf.